spec/sidekiq/evss/disability_compensation_form/submit_form526_all_claim_spec.rb

```diff
@@ -317,15 +319,61 @@ def expect_non_retryable_error
         submission.reload
 
         classification_codes = submission.form['form526']['form526']['disabilities'].pluck('classificationCode')
-        expect(classification_codes).to eq([9012, 8994, nil])
+        expect(classification_codes).to eq([9012, 8994, nil, nil])
       end
 
-      it 'calls va-gov-claim-classifier' do
+      it 'calls va-gov-claim-classifier as default' do
+        vro_client_mock = instance_double(VirtualRegionalOffice::Client)
+        allow(VirtualRegionalOffice::Client).to receive(:new).and_return(vro_client_mock)
+        allow(vro_client_mock).to receive_messages(
+          classify_vagov_contentions_expanded: OpenStruct.new(body: 'expanded classification'),
+          classify_vagov_contentions: OpenStruct.new(body: 'regular response')
+        )
+
+        expect_any_instance_of(Form526Submission).to receive(:classify_vagov_contentions).and_call_original
+        expect(vro_client_mock).to receive(:classify_vagov_contentions)
         subject.perform_async(submission.id)
-        expect_any_instance_of(Form526Submission).to receive(:classify_vagov_contentions)
         described_class.drain
       end
 
+      context 'when the expanded classification endpoint is enabled' do
+        before do
+          user = OpenStruct.new({ flipper_id: submission.user_uuid })
+          Flipper.enable(:disability_526_expanded_contention_classification, user)
+        end
+
+        after do
+          Flipper.disable(:disability_526_expanded_contention_classification)
```

Comment:
Can you mock these? since our tests run in parallel in CI, enabling/disabling them like this can lead to some intermittent failures. 

It would be like this for disabling:
```ruby
allow(Flipper).to receive(:enabled?).with(:feature_flag).and_return(false)
```
We recently updated [the docs](https://depo-platform-documentation.scrollhelp.site/developer-docs/feature-toggles-guide#FeatureTogglesGuide-Backendexample) to show this example.